src/Stryker.Core/Stryker.Core/Helpers/RoslynHelper.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Stryker.Core.Mutants;
+
+namespace Stryker.Core.Helpers
+{
+    internal static class RoslynHelper
+    {
+        // gets the return type of a method (incl. constructor, destructor...)
+        public static TypeSyntax ReturnType(this BaseMethodDeclarationSyntax baseMethod)
+        {
+            return baseMethod switch
+            {
+                ConstructorDeclarationSyntax _ => null,
+                ConversionOperatorDeclarationSyntax _ => null,
+                DestructorDeclarationSyntax _ => null,
+                MethodDeclarationSyntax method => method.ReturnType,
+                OperatorDeclarationSyntax oper => oper.ReturnType,
```

Comment:
```suggestion
                OperatorDeclarationSyntax operator => operator.ReturnType,
```

It's not that long ;)

Reply:
actually, you cannot use 'operator', as it is a keyword. I ended up using operatorSyntax (could have used @operator, but I think it could have been misleading). Anyway, this is really a trivial helper

Reply:
ah right